source/extensions/tracers/opencensus/opencensus_tracer_impl.cc

```diff
@@ -192,6 +194,9 @@ void Span::log(SystemTime /*timestamp*/, const std::string& event) {
   // timestamp is ignored.
   span_.AddAnnotation(event);
 }
+void Span::setBaggage(const std::string&, const std::string&) {}
+
+std::string Span::getBaggage(const std::string&) { return std::string(); }
```

Comment:
consider logging a warning here that this tracer doesn't support baggage (and likewise for the other tracers that do not support it)

Reply:
you could even log an error saying baggage isn't supported similar to https://github.com/envoyproxy/envoy/blob/68df9d17348fea587784da55d6e6aa8c3b06d743/source/extensions/tracers/opencensus/opencensus_tracer_impl.cc#L358

Reply:
A warning isn't safe in the data path. I would either do nothing and leave a TODO or have a stat for dropped baggage. Same elsewhere.

Reply:
I left comments for the spans that don't support baggage (opencensus, xray) and a TODO for zipkin